src/resource-manager.c

```diff
@@ -1106,8 +1106,9 @@ command_special_processing (ResourceManager *resmgr,
                             Tpm2Command     *command)
 {
     Tpm2Response *response   = NULL;
+    TPM2_CC cc = tpm2_command_get_code (command);
 
-    switch (tpm2_command_get_code (command)) {
+    switch (cc) {
```

Comment:
Don't change this hunk, lets keep the patch as small as possible. and not modify the switch at all.